src/start/PasswordAuthView.js

```diff
@@ -28,9 +37,31 @@ class PasswordAuthView extends React.Component {
       progress: false,
       email: props.email || config.defaultLoginEmail,
       password: props.password || config.defaultLoginPassword,
+      onepasswordHidden: false,
     };
   }
 
+  componentWillMount() {
+    OnePassword.isSupported()
+      .then(
+        () => {
+          console.log('OnePassword app is installed and ready!');
+          this.setState({ onepasswordHidden: false });
+        },
+        () => {
+          console.log('OnePassword not available.');
+          this.setState({ onepasswordHidden: true });
+        }
+      );
+  }
+
+  findOnepasswordLogin = () => {
+    OnePassword.findLogin('@https://chat.zulip.org')
```

Comment:
For the URL you'll want to pass `realm` as prop to `PasswordAuthView` from `AuthScreen`